crates/router/src/connector/bluesnap.rs

```diff
@@ -179,6 +180,7 @@ impl
         req: &types::ConnectorCustomerRouterData,
     ) -> CustomResult<Option<String>, errors::ConnectorError> {
         let connector_request = bluesnap::BluesnapCustomerRequest::try_from(req)?;
+        router_env::logger::info!(?connector_request);
```

Comment:
router_env::logger::info!(connector_request=?connector_request);

Reply:
not required this will actually use the keyword connector_request